Content.Client/EntryPoint.cs

```diff
@@ -149,7 +150,12 @@ public override void PostInit()
             IoCManager.Resolve<IGameHud>().Initialize();
             IoCManager.Resolve<IClientNotifyManager>().Initialize();
             IoCManager.Resolve<IClientGameTicker>().Initialize();
-            IoCManager.Resolve<IOverlayManager>().AddOverlay(new ParallaxOverlay());
+            var overlayMgr = IoCManager.Resolve<IOverlayManager>();
+            overlayMgr.AddOverlay(new ParallaxOverlay());
+            overlayMgr.AddOverlay(new GradientCircleMaskOverlay());
+            overlayMgr.AddOverlay(new CircleMaskOverlay());
+            overlayMgr.AddOverlay(new FlashOverlay());
+            overlayMgr.AddOverlay(new RadiationPulseOverlay());
```

Comment:
Seems icky? Would be better for relevant systems to handle adding and removing overlays as needed.

Reply:
That's what OverlayEffectsComponent did. The result was everything getting out of sync and causing bugs.